S2VX.Game.Tests/VisualTests/GameHoldNoteTests.cs

```diff
@@ -52,8 +125,137 @@ public class GameHoldNoteTests : S2VXTestScene {
             AddUntilStep("Wait until all notes are removed", () => Story.Notes.Children.Count == 0);
             AddStep("Release key", () => InputManager.ReleaseKey(Key.Z));
             AddAssert("Does not trigger multiple notes", () =>
-                Precision.AlmostEquals(originalNoteCount * 100, PlayScreen.ScoreInfo.Score, 20)
+                Precision.AlmostEquals((originalNoteCount - 1) * 100, PlayScreen.ScoreInfo.Score, 20)
```

Comment:
Since score is more exact now, can we remove using `AlmostEquals`?

Reply:
Done